Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -281,47 +281,57 @@ router.beforeEach(async (to, from, next) => {
     );
     if (to.meta.routeIsOidcCallback || to.meta.stateless) {
         next();
-    } else {
-        // Make sure that the route accepts the current state
-        store.dispatch("auth/oidcCheckUser").then((isValid: boolean) => {
-            logger.info("User is valid: " + isValid);
+        return;
```

Comment:
looks like sonar really likes 'early returns' vs 'elses'

Reply:
Yeah.  Could be related to reducing nesting, which can be nice.

Reply:
the smell was "code complexity too high" which i was able to mitigate by reducing the nesting! there would be other ways of refactoring it, I personally like early returns than nesting